.github/workflows/backport.yml

```diff
@@ -23,18 +23,20 @@ jobs:
         with:
           fetch-depth: 0
 
+      # This is necessary as the repository is checked out to avoid merge conflicts
+      # on daily canary file
+      - name: Sets 
+        run: |
+          git config --global user.name "github-actions[bot]"
```

Comment:
I thought it'd be nice to have this committed by the existing CCF bot but the backport PR is already authored by the GH Actions bot so making this consistent here,